api/permissions.py

```diff
@@ -9,9 +10,9 @@ class IsAllowedForAction(permissions.BasePermission):
     def has_permission(self, request, view):
         key = request.META["HTTP_AUTHORIZATION"].split()[1]
```

Comment:
maybe use a .get here, to prevent from having a KeyError. Was previously used in combination with HasAPIKey which was checking this.